### ISSUE TITLE
Download images from multiple sites instead of multiple images from one site

### ISSUE THREAD

**Issue by Gradivus500** [Bionus/imgbrd-grabber#1795]
**Is your feature request related to a problem? Please describe**

Download images from 1 to 10 sites at a time instead of downloading 1 to 10 images from a single site.

I think this is better for both the user and site owners. The user doesn't run the risk of getting throttled by the website, and instead of a single website getting hit hard by requests, many websites only get hit with a few.

**Comment by RiabikovR** [comment]
Exactly what I've been thinking every time I got throttled by pixiv/sankaku.

**Comment by Bionus** [comment]
Is this in the context where you download multiple batches at once?
Because when you download a single batch of images, I don't see how that's possible.

**Comment by RiabikovR** [comment]
Yeah, when downloading from multiple sources.

**Comment by Gradivus500** [comment]
Yes, I am exclusively referring to when you have multiple sources in a download batch.

**Comment by TeKett** [comment]
Well there is sorta a solution for the moment, tho only for us users. What you do is use the "-grabber:downloaded" filter and first download from gelbooru or rule34xxx, as that is insanely fast (and i know, not so nice towards them but), then download from the rest, and set sankaku and pixiv last. When you get to sankaku and pixiv 90-99% of all images with the same MD5 will already have been downloaded and gets skipped _before_ the download instead of _during_ download so they dont count towards the throttle. I know there is some manual work since sankaku gets places above quite a few sources, but it works, and most users dont try and download a couple million posts over thousands of artists at the same time.

**Comment by Gradivus500** [comment]
I think this feature request is still relevant.
